lib/impure/db_mysql.nim

```diff
@@ -150,25 +150,25 @@ proc tryExec*(db: DbConn, query: SqlQuery, args: varargs[string, `$`]): bool {.
   tags: [ReadDbEffect, WriteDbEffect].} =
   ## tries to execute the query and returns true if successful, false otherwise.
   var q = dbFormat(query, args)
-  return mysql.realQuery(PMySQL db, q, q.len) == 0'i32
+  return mysql.real_query(PMySQL db, q, q.len) == 0'i32
```

Comment:
I would like to use camelCase, however the whole file of mysql.nim uses snake_case. So I decide to live with it now.

Reply:
what we've done in such cases is to separate the *pure* C API into a private module that does nothing else than `importc`, then add a wrapper on top that makes the wrapper convenient to use from nim - this works pretty well because it also separates the responsibilities of the various modules along a clean line